### DISCUSSION TITLE
pimcore-install stuck at step 6

### DISCUSSION THREAD

**Discussion by Cersch** [pimcore/pimcore#10412] [Q&A / Support]
![image](https://user-images.githubusercontent.com/87798605/134667155-f0cc8bd4-07aa-4127-a4fa-1f7938adc4ef.png)

![image](https://user-images.githubusercontent.com/87798605/134667229-80ccbda8-11ce-4200-8b2a-460a75218f34.png)

Why am I stuck here? Database is working and tables are being generated by the installer..

Pimcore 10.1. something 
Php 8.0
Mysql 8.0

Running in Docker Container

var/log/dev.log -> 

![image](https://user-images.githubusercontent.com/87798605/134667727-894cacf5-3867-40ac-bb21-4064583b9fc6.png)

**Comment by mzcentric** [comment]
Are you using windows 10 or linux...
For Linux install apache + php (>=8.0) + mysql + phpmyadmin (google it and you find tutorials for installing them)
1. Then install composer with version 2.x or above from composer site 
2. Then edit the php.ini file inside apache folder and set memory_limit = -1 (this will unlimited memory as composer require it)
3. Save php.ini and restart apache server
4. Then use the COMPOSER_MEMORY_LIMIT=-1 composer create-project pimcore/skeleton my-project (if you want to install skeleton or refer to pimcore installation manual for demo)
5. Once project is created enter into it via termial with say /var/www/project-name
6. inside this folder inside terminal type ./vendor/bin/pimcore-install command
7. Meanwhile when the command is executing open a new terminal and fire database creation command from pimcore installation manual
8. Once installation on previous terminal is near to finish it will ask for admin name password and database related credentials provide it
9. After installation finishes you need to update apache virtual hosts and hosts file of linux to setup virtual host so you can access it via browser, reason is that if you access it via lets say http://127.0.0.1/project-name OR http://localhost/project-name it will not load assets as they are all mapped to load from root level

**Comment by mzcentric** [comment]
For Windows install xampp
1. Then install composer with version 2.x or above from composer site 
2. Then edit the php.ini file inside xampp ->apache->php folder and set memory_limit = -1 (this will unlimited memory as composer require it)
3. Save php.ini and restart xampp server
4. Then use the composer create-project pimcore/skeleton my-project (if you want to install skeleton or refer to pimcore installation manual for demo)
5. Once project is created enter into it via UI that C:\xampp\htdocs folder
6. inside this folder Open terminal and type .\vendor\bin\pimcore-install command (keep an eye on forward slash as windows do not know this -> /)
7. Meanwhile when the command is executing open a new terminal and open phpmyadmin on browser and create database from interface
8. Once installation on terminal is near to finish it will ask for admin name password and database related credentials provide it
9. After installation finishes you need to update apache virtual hosts and hosts file of Windows to setup virtual host so you can access it via browser, reason is that if you access it via lets say http://127.0.0.1/project-name OR http://localhost/project-name it will not load assets as they are all mapped to load from root level

**Reply by mzcentric** [reply]
I saw the error log picture.. is your database charset=utf8mb4; or not since logs point to it

**Reply by Cersch** [reply]
Thank you for your very detailed Answer. 

1. It is centOs
2. installation succeede
3. Database is mysql 8
4. php > 8
5. Memory 1024M
6. nginx container -> tcp/9000 -> docker container with pimcore & php 
7. getting bad gateway 502
8. tried the nginx config from pimcore website .. edited it for my server -> not working -> #

[site.txt](https://github.com/pimcore/pimcore/files/7335728/site.txt)
d

**Reply by mzcentric** [reply]
make these folder writable with
         a. sudo chmod -R 777 /var
         b. sudo chmod  -R 777 /var/www
         c. sudo chmod  -R 777 /var/www/project-name/var
then cd into project-name and inside the project-name folder open terminal and execute following command
         1. php bin/console cache:clear

**Reply by mzcentric** [reply]
please refer to your OS docs if any confusion arise regarding folders location

**Reply by IkennaNdukwe** [reply]
@mzcentric 

Hello, I was able to follow the steps provided successfully except no. 9. 
After updating both the apache virtual hosts and host file of windows, i still get the "404 Not Found" error. 

Please see attached screenshots of the conf files.

Thanks.

![image](https://user-images.githubusercontent.com/91786168/152692102-64e0c0ce-4433-4da8-a6cc-a7b191bf6544.png)
![image](https://user-images.githubusercontent.com/91786168/152692174-3695e991-167b-45b2-803c-dfa4d0a6ce13.png)
